src/dnx.coreclr.unix/tpa.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+BOOL CreateTpaBase(LPWSTR** ppNames, size_t* pcNames, bool bNative);
```

Comment:
The existing version of the file already does the same thing.  They assume that windows.h and friends will already be included.